### ISSUE TITLE
Add exercise(s) for the branching episode

### ISSUE THREAD

**Issue by SaranjeetKaur** [ImperialCollegeLondon/intermediate_grad_school_git_course#63]
This episode should probably have an exercise (address later)

_Originally posted by @AdrianDAlessandro in https://github.com/ImperialCollegeLondon/intermediate_grad_school_git_course/pull/59#discussion_r1790427302_

**Comment by alexdewar** [comment]
Does this still apply after the (second) reorganisation of the branching + merging episodes?

**Comment by AdrianDAlessandro** [comment]
No longer needed after #78
